app/src/main/java/com/maciejmalak/githubstatistics/presenters/PresenterImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.maciejmalak.githubstatistics.presenters;
+
+import com.maciejmalak.githubstatistics.activities.HomeView;
+import com.maciejmalak.githubstatistics.api.GithubInteractor;
+import com.maciejmalak.githubstatistics.api.GithubInteractorImpl;
+import com.maciejmalak.githubstatistics.model.Owner;
+
+public class PresenterImpl implements Presenter, GithubInteractor.OnFinishedListener {
+    private HomeView mCurrentView;
+    private GithubInteractorImpl mGithubInteractorImpl; /* TODO interface type instead of concrete class*/
+
+    public PresenterImpl(final HomeView view) {
+        mCurrentView = view;
+        mGithubInteractorImpl = new GithubInteractorImpl(this);
```

Comment:
I think it's better to prepare separate methods for view binding (bindView, unbindView), consider situation where presenter is injected or is kept as a singleton through whole activity lifecycle - it may cause memory leaks. I attached sample classes.
`public interface Presenter<T> {
    void attachView(T view);

    void detachView();
}

public abstract class AbstractPresenter <T>implements Presenter<T> {

    T mView;

    @Override
    public void attachView(final T view) {
        this.mView = view;
    }

    @Override
    public void detachView() {
        mView = null;
    }
}
`
T is a view class. I usually also use T extends MvpView to make a constraints for generic type.

Regarding to all services/interactors/dao/etc I would pass them by using constructor instead of creating instance directly in cstr. Such approach prepares code for unit tests and possible mocks of services.